### PR TITLE
Case 22054: flat models can introduce nan into geometry

### PR DESCRIPTION
**Pull request by AndrewMeadows**
https://highfidelity.manuscript.com/f/cases/22054/flat-mesh-models-can-introduce-nan-into-final-geometry

This PR fixes a bug that prevents some models from rendering

### CODE REVIEW

File: libraries/render-utils/src/Model.cpp [hunk 1]
```diff
@@ -1289,6 +1289,8 @@ void Model::scaleToFit() {
     // size is our "target size in world space"
     // we need to set our model scale so that the extents of the mesh, fit in a box that size...
     glm::vec3 meshDimensions = modelMeshExtents.maximum - modelMeshExtents.minimum;
+    const glm::vec3 MIN_MESH_DIMENSIONS { 1.0e-6f };  // one micrometer
+    meshDimensions = glm::max(meshDimensions, MIN_MESH_DIMENSIONS);
```
Comment (AndrewMeadows) [inline libraries/render-utils/src/Model.cpp]: This is the fix.

File: libraries/render-utils/src/Model.h [hunk 2]
```diff
@@ -397,7 +397,7 @@ public slots:
 
     glm::vec3 _translation; // this is the translation in world coordinates to the model's registration point
     glm::quat _rotation;
-    glm::vec3 _scale;
+    glm::vec3 _scale { 1.0f };
```
Comment (AndrewMeadows) [inline libraries/render-utils/src/Model.h]: This is not a fix, but it expresses the default expectation for **Model::_scale** in an implict initialization.  The **only** other place where `Model::_scale` is set is in `Model::setScaleInternal()`.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18573/HighFidelity-Beta-PR15324-72ac35c.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18573/HighFidelity-Quest-Beta-PR15324-72ac35c.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18573/HighFidelity-Beta-PR15324-72ac35c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18573/HighFidelity-Beta-Interface-PR15324-72ac35c.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18573/HighFidelity-Beta-PR15324-72ac35c.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18573/HighFidelity-Beta-Interface-PR15324-72ac35c.exe

**Pull request merged**